core/modules/simpletest/tests/common.test

```diff
@@ -1095,6 +1095,10 @@ class CommonBackdropHTTPRequestTestCase extends BackdropWebTestCase {
     $redirect_301 = backdrop_http_request(url('system-test/redirect/301', array('absolute' => TRUE)), array('max_redirects' => 0));
     $this->assertFalse(isset($redirect_301->redirect_code), 'backdrop_http_request does not follow 301 redirect if max_redirects = 0.');
 
+    $redirect_invalid = backdrop_http_request(url('system-test/redirect-protocol-relative', array('absolute' => TRUE)), array('max_redirects' => 1));
+    $this->assertEqual($redirect_invalid->code, -1002, format_string('301 redirect to protocol-relative URL returned with error code !error.', array('!error' => $redirect_invalid->error)));
```

Comment:
```suggestion
    $this->assertEqual($redirect_invalid->code, -1002, format_string('301 redirect to protocol-relative URL returned with error code !error.', array('!error' => $redirect_invalid->code)));
```

Reply:
Good catch. I'll also fix the rest of the assertions in that test with similar problems..